### PR TITLE
Add standard SUPPORT.md file

### PR DESCRIPTION
**Pull request by theunrepentantgeek**
**What this PR does / why we need it**:

The Microsoft Open Source Program Office is recommending the use of a `SUPPORT.md` file to make it easy for users to see what level of support each repo provides.

I've taken their supplied template and customized it with information already present in our `README.md`.

**Special notes for your reviewer**:

Are there any other support resources/routes that should be included?

**How does this PR make you feel**:
gif

**If applicable**:
- [x] this PR contains documentation

### CODE REVIEW

**Review by Porges — DISMISSED** [review]

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Support
+
+## How to file issues and get help
+
+This project uses GitHub Issues to track bugs and feature requests. Please [search](https://github.com/Azure/azure-service-operator/issues) the existing issues before filing new issues to avoid duplicates. For new issues, file your bug or feature request as [a new issue](https://github.com/Azure/azure-service-operator/issues/new/choose). The Azure Service Operator project maintainers will respond to the best of their abilities.
```
Comment (matthchr) [inline SUPPORT.md:5]: Should we remove this information from the read me (and replace with a section linking here?)

Just reducing document duplication?

Also I see the README says:
> Azure Service Operator is an open source project that is **not** covered by the [Microsoft Azure support policy](https://support.microsoft.com/en-us/help/2941892/support-for-linux-and-open-source-technology-in-azure). 

Should be include that here too?

**Review by matthchr — CHANGES_REQUESTED** [review]
See comment about reducing document duplication

**Review by babbageclunk — DISMISSED** [review]
I like Matt's suggestion about pointing the readme at this too.

File: SUPPORT.md (hunk 1 above)
Comment (theunrepentantgeek) [reply SUPPORT.md:5]: I've added that link to `SUPPORT.md`.

Won't remove the duplication from the `README.md` as the goal is to make support details more discoverable, not to require people to look in one specific place.

**Review by matthchr — APPROVED** [review]

**Pull request merged**